viewing_party/main.py

```diff
@@ -0,0 +1,227 @@
+#************** WAVE 1 ***************
+
+def create_movie(title, genre, rating):
+    
+    new_movie = {}
+    
+    if type(title) == str and type(genre) == str and type(rating) == float:
+        new_movie["title"] = title 
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+
+        return new_movie
+
+    else: 
+        return None
```

Comment:
You don't need to check if inputs are strings, just that they aren't `None`.

Also, you can clean this up a little using a dictionary literal:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
    else: 
        return None
```